src/models/Pools.ts

```diff
@@ -20,3 +20,42 @@ export class SovrynPoolSchema {
   @prop()
   lastUpdatedAt?: Date;
 }
+
+@index({chainId: 1, protocol: 1, token0: 1, token1: 1})
+@index({chainId: 1, fee: 1, token0: 1, token1: 1, address: 1}, {unique: true})
+@index({chainId: 1, address: 1})
+@modelOptions({options: {allowMixed: Severity.ALLOW}})
+export class Pool {
+  @prop({required: true})
+  chainId!: string;
+
+  @prop({required: true})
+  protocol!: string;
+
+  @prop({required: true})
+  token0!: string;
+
+  @prop({required: true})
+  token1!: string;
+
+  @prop({required: true})
+  fee!: number;
+
+  @prop({required: true})
+  address!: string;
+
+  @prop()
+  liquidityActive!: number;
+
+  @prop({required: true})
+  liquidityLockedToken0!: number;
+
+  @prop({required: true})
+  liquidityLockedToken1!: number;
```

Comment:
Would it be `lockedLiquidityTokenX` better?

Reply:
I mean not `liquidityLockedToken1` but `lockedLiquidityToken1`, just grammar.

Reply:
let's use the same names as in uniswap example